pkgs/development/python-modules/matrix-nio/default.nix

```diff
@@ -39,6 +40,21 @@ buildPythonPackage rec {
     hash = "sha256-hFSS2Nys95YJgBNED8SBan24iRo2q/UOr6pqUPAF5Ms=";
   };
 
+  patches = [
+    # Those two patches makes `matrix-nio` more compatible with alternative
+    # Matrix homeservers like conduit.rs. They are critical for… well using this library…
+    # https://github.com/matrix-nio/matrix-nio/pull/483
+    (fetchpatch {
+      url = "https://github.com/matrix-nio/matrix-nio/commit/5c1a393fa16d5310e458539d64c25e0c2034a025.patch";
+      hash = "sha256-C39eufMI/HITnElEBpj0jBUIn3m0/XxXdxRdz1fhXhs=";
+    })
+    # https://github.com/matrix-nio/matrix-nio/pull/482
+    (fetchpatch {
+      url = "https://github.com/matrix-nio/matrix-nio/pull/482/commits/64c646edb918fe7ef57a6f9d76d95c24d3c41c07.patch";
+      hash = "sha256-SllZV+gBMjqfX9BUD1OrmZpy617c2FO110As9hw5mS0=;";
+    })
```

Comment:
Given how new these PRs are, I think it makes sense to give upstream a few days to review and merge instead of patching? (Then, if they don't plan to cut a new release any, keep the patch)

Reply:
Yes, this PR can stay open for few days and then if something changes, I can remove it, if nothing change, I can do nothing. :)

Reply:
note that fetching commits in PRs can result in 404 if you force push the PR (for example after a review)